packages/schemas/src/models/tenants.ts

```diff
@@ -31,8 +31,11 @@ export const Tenants = createModel(
 
 export type TenantModel = InferModelType<typeof Tenants>;
 
-export type TenantInfo = Pick<TenantModel, 'id' | 'name' | 'tag'> & { indicator: string };
+export type TenantInfo = Pick<TenantModel, 'id' | 'name' | 'tag'> & {
+  indicator: string;
+  isFreePlan: boolean;
```

Comment:
How about use `planId`, we have free id constants.

Reply:
I think we only need to distinguish whether the tenant is a free tenant. If we use `planId`, we need to do the check on the console side. I prefer to have this tag returned by the backend API.

Reply:
`planId` would be helpful if @xiaoyijun wants to render a tenant tag like: free, hobby, pro, ...

Reply:
So true, let me update this.